src/SamplesServiceModelGenerator.Tests/Swagger/ParserTests.cs

```diff
@@ -0,0 +1,141 @@
+﻿using FluentAssertions;
+using Newtonsoft.Json.Linq;
+using NUnit.Framework;
+using SamplesServiceModelGenerator.Swagger;
+using Property = SamplesServiceModelGenerator.Swagger.Property;
+
+namespace SamplesServiceModelGenerator.Tests.Swagger
+{
+    [TestFixture]
+    public class ParserTests
+    {
+        const string _baseUrl = "https://demo.aqsamples.com/api/swagger.json";
+        private string _jsonText = "";
+
+        private Parser _testparser = new Parser();
+        private Api _parseOutput = new Api();
+        private JObject _json = new JObject();
+
+        private string LoadStringFromUrl(string url)
+        {
+            var uri = new Uri(url);
+
+            Console.WriteLine($"Fetching {uri} ...");
+
+            using (var client = new HttpClient())
+            {
+                var response = client.GetAsync(uri).Result;
+                return response.Content.ReadAsStringAsync().Result;
+            }
+        }
+
+        [OneTimeSetUp]
+        public void ForEachTest()
+        {
+            _jsonText = LoadStringFromUrl(_baseUrl);
+
+            try
+            {
+                _json = JObject.Parse(_jsonText);
+            }
+            catch (Exception ex)
+            {
+                throw new Exception("Test Swagger is not a Valid Json", ex);
+            }
+            _parseOutput = _testparser.Parse(_jsonText, _baseUrl);
+        }
+
+        [Test]
+        public void ParseBaseUrl()
```

Comment:
It's sort of awkward since all of these tests are just assertions of the same test case (ie Can `Parser` consume the json doc returned by Samples's dev server). But would be nice to still follow standard naming practices
```suggestion
        public void Parse_SamplesSwaggerDoc_ParsesBaseUrl()
```